### PR TITLE
kubectl quick reference document

### PR DESCRIPTION
**Pull request by alkar**

### CODE REVIEW

**Comment by alkar** [comment]
I'm not sure that `{:toc}` will work.

File: 03-other-topics/003-quick-reference.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Kubernetes quick reference
+
+This document acts as a quick reference to `kubectl`, listing some of the most common operations.
+
+The examples here only address the most basic approach to these operations. For more options, please refer to the command-line help of `kubectl` subcommands.
+
+There is also a more detailed [cheatsheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/) in the official kubernetes documentation.
+
+## Table of contents
+{:toc}
```
Comment (kalbir) [inline 03-other-topics/003-quick-reference.md]: Yeah this doesnt work. You can add any aribtrary list immediately before it to make it work:

```
* TOC
{:toc}
```

That works.

*However* you also get a table of contents autogenerated in the left panel of the page, so do you need two of them?

screen shot 2018-07-12 at 16 52 50

Comment (kalbir) [reply 03-other-topics/003-quick-reference.md]: ...and as I can now see from looking at the screenshot (isn't it weird how looking at a screenshot is so different to looking at your _screen_?) the table of contents that is generated contains the page title and `table of contents` as a heading. You can remove this by adding a `no_toc` class to the heading, which in `Kramdown` you do like the following:

```
# Kubernetes quick reference
{:.no_toc}

```

⬆️ that will stop "Kubernetes quick reference" showing up.

Comment (kalbir) [reply 03-other-topics/003-quick-reference.md]: Actually, turns out that in turn breaks the way the side contents works 
screen shot 2018-07-12 at 17 03 16

So I'm even less convinced we need the contents inline (I don't know how to fix this problem!)

**Review by kalbir — CHANGES_REQUESTED** [review]
The table of contents doesn't work (I've shown how to fix it) but do we actually need it?

**Comment by alkar** [comment]
We definitely do not need an inline TOC - I just forgot that we get the sidebar TOC by default. Removed.

**Review by kalbir — APPROVED** [review]
Yep that's better without the table of contents

**Pull request merged**